### ISSUE TITLE
Dialog (Modal) Pattern. A question of modality and focus management

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/aria-practices#2924]
I am seeking guidance to understand the tab ring behaviour for non-modal dialogs that use aria-modal false. This is specifically in reference to the ARIA Practices Guide for [Dialog (Modal) ](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/#:~:text=Like%20non%2Dmodal,closing%20the%20dialog.)

_"Like non-modal dialogs, modal dialogs contain their tab sequence. That is, Tab and Shift + Tab do not move focus outside the dialog. However, unlike most non-modal dialogs, modal dialogs do not provide means for moving keyboard focus outside the dialog window without closing the dialog."_

For a non-modal dialog, for example an information sheet, that is non-critical, how should this work. What recommended practices are there for the keyboard user to interact with the main screen without dismissing the modal. Would being able to minimize the dialog be an accepted means to enable access to the main page content. Would a button that the user can click to move the use to the main  page content be sufficient.

What is the recommendation for focus management when the user is focused in the main content and how do they return to the dialog? Is the only way to access the dialog content clicking the triggering button?

I have these questions swirling around in my head and seek clarity in what recommendations to make in regards to accessibility for a non-modal dialog.

Thanks in advance.

**Comment by LaurenceRLewis** [comment]
[Keyboard method to jump out of NON-modal dialogs? #599](https://github.com/w3c/aria-practices/issues/599)
